addons/website_cf_turnstile/static/src/js/turnstile.js

```diff
@@ -19,15 +19,29 @@ publicWidget.registry.s_website_form.include({
                          data-response-field-name="turnstile_captcha"
                          data-sitekey="${session.turnstile_site_key}"
                          data-error-callback="throwTurnstileError"
+                         data-before-interactive-callback="TurnstileBeforeInteractive"
+                         data-after-interactive-callback="TurnstileAfterInteractive"
                     ></div>
                     <script class="s_turnstile">
-                        // Rethrow the error, or we only will catch a "Script error" without any info 
+                        // Rethrow the error, or we only will catch a "Script error" without any info
                         // because of the script api.js originating from a different domain.
                         function throwTurnstileError(code) {
                             const error = new Error("Turnstile Error");
                             error.code = code;
                             throw error;
                         }
+                        function TurnstileBeforeInteractive() {
+                            const btn = document.querySelector('.s_website_form_send, .o_website_form_send');
```

Comment:
`btnEl` ?
Always use double quotes in Javascript.

Reply:
btnEl ? not sure what btn could be else...

`'`vs `"` is realy discutable. From last guidelines I follow, `'`for technical string (like selector) and `"` for 'end user' string